jetty-util/src/main/java/org/eclipse/jetty/util/MultiMap.java

```diff
@@ -60,14 +60,25 @@ public MultiMap(MultiMap<V> map)
         return vals;
     }
 
+    /**
+     * Get the first value from a multiple value.
+     *
+     * @param name The entry key.
+     * @return Unmodifieable List of values.
+     */
+    public V getValue(String name)
+    {
+        return getValue(name, 0);
+    }
+
```

Comment:
Do we still need this class at all? Feels like with modern JDK `Map` with `compute()`, `merge()`, `flatMap()`, etc we can get rid of this class?

Reply:
+1

Reply:
Yeah it feels very out of date.   But can we punt to a different PR to consider replacing it.  Would be best to do when everything is ported and compiling again so we can refactor all at once.  I will add a TODO.